### DISCUSSION TITLE
tuiNotificationOptionsProvider throwing compiler errors

### DISCUSSION THREAD

**Discussion by loxy** [taiga-family/taiga-ui#9525] [Q&A] [answered]
Hi!

when I try to configure alerts / notifications I get compiler errors:

```
TS2353: Object literal may only specify known properties, and 'autoClose' does not exist in type 'Partial<TuiNotificationOptions>'
```

I tried in Stackblitz. Same: https://stackblitz.com/edit/angular-chgduv

I just followed the docs on: https://taiga-ui.dev/components/alert/Setup

Are docs outdated? If yes, what is the correct way to specify `autoClose`?

**Comment by waterplea** [comment, marked as answer]
Docs are outdated, alerts now have separate token object that depends on notification options:
```ts
export const TUI_ALERT_OPTIONS = tuiCreateTokenFromFactory<TuiAlertOptions>(() => ({
    ...TUI_ALERT_DEFAULT_OPTIONS,
    ...inject(TUI_NOTIFICATION_OPTIONS),
}));
```
I'll add helper function and update docs.

**Reply by loxy** [reply]
Was there something already added?

**Reply by waterplea** [reply]
Yes:
<img width="1234" alt="image" src="https://github.com/user-attachments/assets/03df73c2-f35e-465a-b4ff-a2451084acf6">
